src/lib/core/selection/pseudo-checkbox/_pseudo-checkbox-theme.scss

```diff
@@ -8,18 +8,19 @@
   $warn: map-get($theme, warn);
   $background: map-get($theme, background);
 
-  // The color of the checkbox's checkmark / mixedmark.
-  $checkbox-mark-color: mat-color($background, background);
-
   // NOTE(traviskaufman): While the spec calls for translucent blacks/whites for disabled colors,
   // this does not work well with elements layered on top of one another. To get around this we
   // blend the colors together based on the base color and the theme background.
   $white-30pct-opacity-on-dark: #686868;
   $black-26pct-opacity-on-light: #b0b0b0;
   $disabled-color: if($is-dark-theme, $white-30pct-opacity-on-dark, $black-26pct-opacity-on-light);
 
-  .mat-pseudo-checkbox::after {
-    color: $checkbox-mark-color;
+  .mat-pseudo-checkbox {
+    color: if($is-dark-theme, rgba(white, 0.7), rgba(black, 0.54));
```

Comment:
Can you use 'secondary-text' from the $foreground palette instead? You won't have to check the theme.